packages/mdc-dialog/mdc-dialog.scss

```diff
@@ -0,0 +1,162 @@
+@import "@material/animation/functions";
+@import "@material/typography/variables";
+@import "@material/elevation/mixins";
+@import "@material/theme/mixins";
+@import "@material/rtl/mixins";
+
+.mdc-dialog {
+  display: flex;
+  position: fixed;
+  align-items: center;
+  justify-content: center;
+  opacity: 0;
+  z-index: -1;
+
+  &__surface {
+    display: inline-flex;
+    position: absolute;
+    bottom: 50vh;
+    flex-direction: column;
+
+    @include mdc-elevation(24);
+
+    width: calc(100% - 30px);
+    min-width: 640px;
+    max-width: 865px;
+    transform: translateY(150px) scale(.8);
+    transition: mdc-animation-enter(opacity, 120ms), mdc-animation-enter(transform, 120ms);
+
+    @include mdc-theme-prop(background-color, background);
+
+    @include mdc-theme-dark(".mdc-dialog") {
+      @include mdc-theme-prop(background-color, text-primary-on-dark);
+    }
+
+    @include mdc-rtl(".mdc-dialog") {
+      text-align: right;
+    }
+
+    font-family: Roboto, sans-serif;
+    opacity: 0;
+  }
+
+  &__backdrop {
+    display: flex;
+    position: fixed;
+    top: 0;
+    left: 0;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 100%;
+    background-color: black;
+    opacity: 0;
+    z-index: -1;
+  }
+
+  &--open {
+    display: flex;
+    top: 0;
+    left: 0;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 100%;
+    opacity: 1;
+    z-index: 2;
+
+    .mdc-dialog__surface {
+      transform: translateY(0) scale(1);
+      transition: mdc-animation-enter(opacity, 120ms), mdc-animation-enter(transform, 120ms);
+      opacity: 1;
+    }
+
+    .mdc-dialog__backdrop {
+      transition: mdc-animation-enter(opacity, 120ms);
+      opacity: .3;
+    }
+  }
+
+  &__header {
+    display: flex;
+    align-items: center;
+    padding: 24px 24px 0;
+
+    &__empty {
+      padding: 0;
+    }
+
+    @include mdc-rtl {
+      text-align: right;
+    }
+  }
+
+  &__header__title {
+    flex: 1;
+    margin: 0;
+    font-size: 20px;
+    font-weight: normal;
+    line-height: 20px;
+  }
+
+  &__body {
+    margin-top: 20px;
+    padding: 0 24px 24px;
+
+    @include mdc-theme-prop(color, text-secondary-on-light);
+
+    @include mdc-theme-dark {
+      @include mdc-theme-prop(color, text-secondary-on-dark);
+    }
+
+    font-size: 16px;
```

Comment:
Please use `rem` for font-size. Also, how come we're not using an `mdc-typography` mixin here? If it's because the spec deviates, then we should add a comment saying so. Also, please use a multiplier for line-height.

```css
font-size: 1rem;
line-height: 1.438;
```